run_scans.py

```diff
@@ -111,7 +111,7 @@ def executeCmd(cmd, dirPath):
     except Exception as e:
         print("Caught exception: {0}".format(e))
     finally:
-        runCommand( ["chmod","-R","g+r",dirPath] )
+        runCommand( ["chmod","-R","g+rw",dirPath] )
```

Comment:
In general (and on the off chance that the permissions didn't exist to do this change) it's safer to set `umask` in the executing environment to handle this